media-sound/qpwgraph/qpwgraph-0.2.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Qt GUI to control the JACK Audio Connection Kit and ALSA sequencer connections"
+HOMEPAGE="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+SRC_URI="https://gitlab.freedesktop.org/rncbc/qpwgraph/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="alsa debug systray"
+
+BDEPEND="virtual/pkgconfig"
+DEPEND="
```

Comment:
The libraries defined in this dependency list have subslots. It's usually a good idea to enable subslot rebuilds, i.e. depend on `cat/pkg:=` or `cat/pkg:slot=`. This ensures that the package is rebuilt if upstream changes the subslot (which usually corresponds to a change in the SONAME version, which requires a rebuild of reverse dependencies).

I heard that there are cases for Qt where subslot rebuilds are not necessary (depending on whether the private ABI is used by the software or not). However, I am not sure about this and if in doubt I suggest enabling the subslot rebuild to be safe.